PokeAlarm/Manager.py

```diff
@@ -373,7 +364,7 @@ def process_pokemon(self, pkmn):
                 log.debug("Pokemon 'cp' was not checked because it was missing.")
 
             # Check the Level of the Pokemon
-            if level != '?':
+            if level is not None and level != 'unkn' and level != '?':
```

Comment:
I was running some code that had cp and level sneak through as None and unkn - but after fixing that problem I forgot to change these monster if statements ... 🤦‍♀️